database/src/pool/postgres.rs

```diff
@@ -254,6 +254,13 @@ static MIGRATIONS: &[&str] = &[
         UNIQUE(aid, component)
     );
     "#,
+    r#"
+    alter table pstat_series rename column cache to scenario;
+    alter table pstat_series rename column statistic to metric;
+    alter table pstat_series add backend text not null default 'llvm';
+    alter table pstat_series drop constraint pstat_series_crate_profile_cache_statistic_key;
+    alter table pstat_series add constraint test_case UNIQUE(crate, profile, scenario, backend, metric);
+    "#,
```

Comment:
Hmm, is this commit renaming some fields as well? Would be good if the commit log mentioned this.

Reply:
Yes. It's specified in a comment among the changes, but now I see that only in the SQLite DB, and you're right that it should also be in the commit message. Thanks.